### PR TITLE
Merge Input component

### PR DESCRIPTION
**Pull request by Hamdrive**
Created:
1. 3 sizes of inputs (sm, md, lg)
2. Input with fully round and round corners
3. Active and mandatory inputs
4. Success and Fail validation input with appropriate colors and text
Input

### CODE REVIEW

**Review by ZuberDunge — COMMENTED** [review]
Well structured code, clean and readable. Everything looks fine,  UI looks good too. Good work Hamza. Good to go from my side.

File: stylesheets/reset.css [hunk 1]
```diff
@@ -65,6 +65,12 @@ svg {
...
 }
 
+input{
+  width: 100%;
+  outline: none;
+  border: none;
+}
+
 /* <---fa-icons---> */
 [class^="f"] {
```
Comment (aryan-13) [inline stylesheets/reset.css:75]: What is this for?

**Review by aryan-13 — COMMENTED** [review]
The code looks good!

File: stylesheets/reset.css (hunk 1 above)
Comment (Hamdrive) [reply stylesheets/reset.css:75]: Setting the default value of font awesome icons, 
using attribute selector I'm looking for class names that start with letter `f`

**Comment by Hamdrive** [comment]
Code reviewed, no changes made. Merging PR to dev

**Pull request merged**